src/components/app/data/services/enterpriseCustomerUser.js

```diff
@@ -204,3 +204,17 @@ export async function updateUserCsodParams({ data }) {
   const url = `${getConfig().LMS_BASE_URL}/integrated_channels/api/v1/cornerstone/save-learner-information`;
   return getAuthenticatedHttpClient().post(url, data);
 }
+
+/**
+ * Helper function to unlink an enterprise customer user by making a POST API request.
+ * @param {string} enterpriseCustomerUserUUID - The UUID of the enterprise customer user to be unlinked.
+ * @param {string} userEmail - The email of the user that should be unlinked.
+ * @returns {Promise} - A promise that resolves when the user is successfully unlinked from the enterprise customer.
+ * @throws {Error} - Throws an error if the unlinking process fails.
+ */
+export async function postUnlinkUserFromEnterprise(enterpriseCustomerUserUUID, userEmail) {
+  const url = `${getConfig().LMS_BASE_URL}/enterprise/api/v1/enterprise-customer/${enterpriseCustomerUserUUID}/unlink_users/`;
+  return getAuthenticatedHttpClient().post(url, { user_emails: [userEmail], is_relinkable: true }).catch((error) => {
```

Comment:
Using `try`/`catch`, similar to other service functions, is preferable over chaining `.catch`.